gamemodes/terrortown/gamemode/client/cl_help/populate_bindings.lua

```diff
@@ -0,0 +1,89 @@
+local materialIcon = Material("vgui/ttt/vskin/helpscreen/bindings")
+
+local function AddBindingCategory(category, parent)
+	local client = LocalPlayer()
+
+	local form = CreateForm(parent, category)
+
+	local bindings = bind.GetSettingsBindings()
+
+	for i = 1, #bindings do
+		local binding = bindings[i]
+
+		if binding.category ~= category then continue end
+
+		local currentBinding = bind.Find(binding.name)
+
+		form:MakeBinder({
+			label = binding.label,
+			select = currentBinding,
+			default = binding.defaultKey,
+			onDisable = function(slf, binder)
+				bind.Remove(currentBinding, binding.name, true)
+
+				binder:SetValue(bind.Find(binding.name))
```

Comment:
Shouldn't the value be always "None" here?

Reply:
probably? It has been this way ever since and I did not bother to investigate further